content/doc/developer/tutorial-adopt/enable-incrementals.adoc

```diff
@@ -0,0 +1,22 @@
+---
+layout: developersection
+title: Enable incrementals
+---
+
+:task-identifier: incrementalify
+:task-description: Publish incremental development artifacts
+
+Jenkins evaluation of pull request builds is faster and easier when incremental builds are enabled. See the jenkins.io link:https://www.jenkins.io/doc/developer/plugin-development/incrementals/[incremental build instructions] for more details.
+
+// Create the branch
+include::doc/developer/tutorial-adopt/_create-a-branch.adoc[]
+
+Run the Maven command to enable incrementals:
```

Comment:
Should explain that this is unnecessary for plugins created from archetype, which are already incrementalified.

Reply:
That's a very good point.  Each of the steps is unnecessary for a plugin that has already completed that step.  Plugins that already have a `Jenkinsfile` do not benefit from the step to add a `Jenkinsfile`.  Plugins that are already incrementalified don't need to be done again.

I wonder if there needs to be specific instructions at the entry of each step so the reader can check if the step is already complete.  If a Jenkinsfile already exists, this step is unnecessary.  If the plugin is already incrementalified, this step is unnecessary.

Reply:
also make sure people commit .mvn, a lot of new devs have ignored .mvn for some reason, and that just messes them up.